spacy/lang/ko/__init__.py

```diff
@@ -31,15 +31,24 @@ def korean_tokenizer_factory(nlp):
 class KoreanTokenizer(DummyTokenizer):
     def __init__(self, vocab: Vocab):
         self.vocab = vocab
-        MeCab = try_mecab_import()  # type: ignore[func-returns-value]
-        self.mecab_tokenizer = MeCab("-F%f[0],%f[7]")
+        self._mecab = try_mecab_import()  # type: ignore[func-returns-value]
```

Comment:
Won't this still raise an `ImportError` when `MeCab` is not installed?

Reply:
No, this will raise an error if `natto-py` is not installed, but it doesn't check for mecab yet. I think most users would prefer to be warned early if their pipeline isn't going to work (with `spacy.blank("ko")`) rather than when they first call the tokenizer.

So this warns early and still lets me run `spacy init vectors ko` with just `natto-py` installed (which is easy to install).

Reply:
It does mean the error message isn't 100% accurate.

Reply:
Right, that makes sense :-) 
So good to merge, or do you want to change anything to the error msg first?

Reply:
No, I can't think of a way to phrase it that wouldn't be even more confusing, and I'm mainly thinking of existing users who are used to the mecab setup. You can still load a `ko` pipeline with a different tokenizer without `natto-py` installed, and `spacy[ko]` includes `natto-py`, so we can point to that if the `init vectors ko` situation if it comes up.